### ISSUE TITLE
no menu items on right click

### ISSUE THREAD

**Issue by erasmion** [kweatherman/Folcolor#10]
Hi,  just trying out folcolor  -  after install i can see the 'color folder' entry in the context menu, however the submenu itself is empty .

thank you for this tool,  installation is great, clean and simple.

**Comment by kweatherman** [comment]
Could be some anti-virus software thinking that Folcolor is malware so it didn't let it write it's simple context menu registry keys?
Is so try white listing it. You can see your self if you look at the simple source that there is nothing malware at all about it.

**Comment by erasmion** [comment]
thanks for coming back.  that's strange, i tried on a machine with no antivirus 

tried again just now, checked the registry - all the entries are there.  this time however i did not even get the menu

however i took a couple of commands from the registry and run them through an autohotkey script and it worked.  so that's still a good way ahead.

one more question if you don't mind, shouldn't open folders also be available (where you see the thumbnail content)?

thanks again

**Comment by kweatherman** [comment]
Must be some kind of edge case. I haven't seen it happen over any install here yet.
Try uninstalling (through the UI) and then reinstalling please. This is should recreate the registry entries.
If you had an antivirus initially, it might have just blocked it the first time.
The entries are static and never updated/checked automatically.

But actually, I didn't test Windows 11 very much yet. What OS(es)?

>>shouldn't open folders also be available (where you see the thumbnail content)?
You are talking about folder views like "Tiles", "Content", etc., that show a sort of half-open folder icon right?
It should be pretty much the same type of view like the default folder options view.
You get different?

**Comment by erasmion** [comment]
thanks for coming back. 

the machine where i tried Folcolor,  is a win10 64bit, no antivirus at all.

install-reinstall is the first thing i tried multiple times, though without luck

registry entries all look good as well

what i have just noticed by accident, is that if i select 2 or more folders, THEN i do get the menu...  however no change occurs if i choose say 'Teal'.    very strange.

i'll keep investigating,  if you never had this bug, might be something on my end - and i will keep you posted if i have news.

regarding the icons, i believe the folder thumbnail is actually made up of two icons.  
one functions as a background, the other is semi-transparent so you can see the contents. 

i usually change the second icon to be fully transparent, that way,  if i have an image called folder.jpg  (ie. album cover),  i can see it without crop.  like below

![covers](https://user-images.githubusercontent.com/121201595/210429903-c94a2465-8f96-44ea-af2b-926404d2d136.png)

you can see both icons in the second column (second and third row) 

![imageres](https://user-images.githubusercontent.com/121201595/210430308-351967fb-96c6-4481-861d-5247ccbb76f3.png)

**Comment by kweatherman** [comment]
Ops, I missed the reply notification.

I just tried changing colors and multiple folders at once and it worked for me.
Using the default settings, or using checkbox selection or anything special?

Sure I don't doubt you have some bug. On another project I did doubt someone because I couldn't reproduce it, but they were right I found out later when I used a new system I built. Just had to be in a similar config for it to happen.

Now I see that. I forgot about those types of views, didn't consider them in the tool, in particular because I usually run everything "details" view. 
Windows will make a a folder "Music", "Videos", instead of "General items" if you drop music and/or video files in a folder.
Some sort of mini preview of selected media files.

I see if you set the color with Folcolor it removes the specialness of the folder.
It probably works like you say, need to have the an additional icon images to cover the little folder overlay part.
I'll have to think about it for a while, and if I even want to add this as a fix/featur.
